index.html

```diff
@@ -3686,46 +3692,42 @@ <h1>Resolution</h1>
 
     <p>
 This section defines the inputs and outputs of <a>DID resolution</a> and <a>DID
-URL dereferencing</a>. These functions are defined in an abstract way. Their
-exact implementation is out of scope for this specification, but some
-considerations for implementors are discussed in [[?DID-RESOLUTION]].
+URL dereferencing</a>.  Their exact implementation is out of scope for this
+specification, but some considerations for implementors are discussed in
+[[?DID-RESOLUTION]].
     </p>
 
     <p>
 All conformant <a>DID resolvers</a> MUST implement the <a>DID resolution</a>
-functions for at least one <a>DID method</a> and MUST be able to return a <a>DID
-document</a> in at least one conformant <a>representation</a>.
+functions for at least one <a>DID method</a> and MUST be able to produce a
+<a>DID document</a> in at least one conformant <a>representation</a>.
     </p>
 
-    <section>
+    <section data-dfn-for="Resolver">
         <h2>DID Resolution</h2>
         <p>
-The <a>DID resolution</a> functions resolve a <a>DID</a> into a <a>DID
-document</a> by using the "Read" operation of the applicable <a>DID method</a>.
-(See <a href="#read-verify"></a>.) The details of how this process is
-accomplished are outside the scope of this specification, but all conformant
-implementations implement two functions which have the following abstract
-forms:
+The <a>DID resolution</a> function resolves a <a>DID</a> into a <a>DID
+document</a> by using the "Read" operation of the applicable <a>DID method</a>
+as described in <a href="#read-verify"></a>. The details of how this process is
+accomplished are outside the scope of this specification, but all conforming
+<a>DID resolvers</a> implement the functions below which have the following
+abstract forms:
         </p>
 
-        <p><code>
-resolve ( did, did-resolution-input-metadata ) <br>
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;-&gt; ( did-resolution-metadata, did-document, did-document-metadata )
-        </code></p>
+        <pre title="Abstract functions for DID Resolution">
+resolve(did, options) →
```

Comment:
do we mean DID or DID URL?

Would prefer to see a note that clarifies the type... and explicitly states if `path`, `query` and `fragment` are allowed.

Reply:
if it's possible to hyperlink to definitions here, that might help avoid any confusion.

Reply:
resolve() takes a DID, not DID URL

Reply:
Both the `resolve*` functions take a DID, the `dereference` function takes a DID URL. This is one of the most important differences between these functions.

Reply:
Agreed, this isn't going to change in this PR.